src/kern/optimized/rvv/kernel.h

```diff
@@ -9,10 +9,6 @@ namespace opt {
 
 void init();
 
-TaskSet llm_embedding_get_int4_float(
```

Comment:
why delete the int4 Embeding？you want it fallback to naive？

Reply:
Yeah, it is not optimized now anyway. I think that we can delete some of these non-optimized functions.